win32/tsk_logical_imager/config.txt

```diff
@@ -0,0 +1,4 @@
+jpg
```

Comment:
Should this file be checked in? I see it's also part of the project. @bcarrier was this intended?

Reply:
Long-term, no it shoudl not be checked in here.  The config file is going to evolve with each step. I don't have a strong opinion about checking it in for now or not.  I'm assuming that when this is done, we'll have a sample config file though, but not in the win32 folder.

Reply:
I'll delete it.